yuhow/Makefile

```diff
@@ -1,76 +0,0 @@
-SHELL := bash
```

Comment:
Actually, it works fine. I now convert all of my figure into EPS format. Everything is OK now.